File: app/src/main/java/com/juniperphoton/myerlist/realm/CacheMigration.java
package com.juniperphoton.myerlist.realm;

import io.realm.DynamicRealm;
import io.realm.RealmMigration;
import io.realm.RealmSchema;

public class CacheMigration implements RealmMigration {
    @Override
    public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
        RealmSchema schema = realm.getSchema();

        if (oldVersion == 1) {

        }
    }
}

File: app/src/main/java/com/juniperphoton/myerlist/realm/CacheMigration.kt
package com.juniperphoton.myerlist.realm

import io.realm.DynamicRealm
import io.realm.RealmMigration

class CacheMigration : RealmMigration {
    override fun migrate(realm: DynamicRealm, oldVersion: Long, newVersion: Long) {
        val schema = realm.schema

        if (oldVersion == 1L) {
        }
    }
}